src/main/java/hobbylist/logic/parser/RateCommandParser.java

```diff
@@ -28,7 +28,11 @@ public RateCommand parse(String args) throws ParseException {
 
         int rating;
         if (argMultimap.getValue(CliSyntax.PREFIX_RATING).isPresent()) {
-            rating = Integer.parseInt(argMultimap.getValue(CliSyntax.PREFIX_RATING).get());
+            try {
+                rating = Integer.parseInt(argMultimap.getValue(CliSyntax.PREFIX_RATING).get());
+            } catch (NumberFormatException e) {
+                throw new ParseException(MESSAGE_INVALID_RATING + RateCommand.MESSAGE_USAGE);
```

Comment:
Instead of displaying the whole command again, it would be better the error message targets specifically the`r/RATING`. 
E.g. 
```
Invalid rating input! 
The rating input should be an integer value between 1 and 5 (inclusive).
```